File: 프로그래머스/unrated/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    cnt = len(t) - len(p) + 1
    ary = []
    
    for i in range(0,cnt):
        cnt = t[i:len(p)+i]
        ary.append(cnt)

    for i in ary:
        if int(i) <= int(p):
            answer += 1

    return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    const cnt = t.length - p.length + 1
    const Pnum = Number(p)
    
    let result = 0
    for(let i=0; i<cnt; i++) {
        let sum = ""

        for(let j=0; j<p.length; j++) {
           sum += t[i+j] 
        }

        if(Number(sum) <= Pnum) {
            result += 1
        }
    }
    return result
}